src/javascript/DesignSystem/DropdownTreeSelect/unchecked.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <style>
+      rect {
+        fill: transparent;
+        stroke: #6D737E;
+      }
+    </style>
+    <rect x="4" y="4" width="16" height="16" rx="1"/>
+</svg>
```

Comment:
We don't have access to this icon by any other ways instead of adding svg files?

Reply:
Because we have to put it into our css via JSS. I cannot import it as a react component :'(